plugins/taofurigana/plugin.js

```diff
@@ -88,92 +88,110 @@ CKEDITOR.plugins.add('taofurigana', {
 			}
 			return false;
 		}
-        /**
-	     * @param {CkEditor} editor - ckEditor instance
-	     */
+		/**
+		 * @param {CkEditor} editor - ckEditor instance
+		 */
 		function furiganaCanBeCreated(editor) {
 			var selection = editor.getSelection();
 			var nativeSelection = selection.getNative();
 
 			return nativeSelection !== null && canInsert(selection)&& isWrappable(nativeSelection);
 		}
 
-	    /**
-	     * @param {Selection} selection
-	     * @returns {boolean}
-	     */
+		/**
+		 * @param {Selection} selection
+		 * @returns {boolean}
+		 */
 		function canInsert(selection) {
 			return !isSelectionEmpty(selection) && selection.getRanges()[0] && !isInFigurana(selection.getRanges()[0].startContainer) ;
 		}
-	    /**
-	     * Change command state according to the current selection content
-	     * @param {CkEditor} editor - ckEditor instance
-	     */
-         function refreshCommandState(editor) {
+		/**
+		 * Change command state according to the current selection content
+		 * @param {CkEditor} editor - ckEditor instance
+		 */
+		function refreshCommandState(editor) {
 			var command = editor.getCommand(commandName);
-            var selection = editor.getSelection();
+      var selection = editor.getSelection();
 
 			if (command) {
 				if (furiganaCanBeCreated(editor)) {
 					command.setState(CKEDITOR.TRISTATE_OFF);
 				} else if (selection.getRanges()[0] && isInFigurana(selection.getRanges()[0].startContainer)) {
-                    command.setState(CKEDITOR.TRISTATE_ON);
-                } else {
+						command.setState(CKEDITOR.TRISTATE_ON);
+				} else {
 					command.setState(CKEDITOR.TRISTATE_DISABLED);
 				}
 			}
 		}
-        // Create the command that can be used to apply the style.
-        editor.addCommand(commandName, {
-            exec: function (editor) {
-                var config = editor.config.taoQtiItem,
-                    selection = editor.getSelection(),
-                    curRange = selection.getRanges()[0],
-                    startNode = curRange.startContainer,
-                    rubyElement,
-                    rbElement,
-                    rtElement,
-                    range,
-                    emptyElement;
+		// Create the command that can be used to apply the style.
+		editor.addCommand(commandName, {
+			exec: function (editor) {
+				var config = editor.config.taoQtiItem,
+					selection = editor.getSelection(),
+					curRange = selection.getRanges()[0],
+					startNode = curRange.startContainer,
+					rubyElement,
+					rbElement,
+					rtElement,
+					range,
+					emptyElement,
+					rbHtml;
+				if (isInFigurana(startNode)) {
+					rubyElement = startNode.getAscendant('ruby');
+					console.log('rubyElement ', rubyElement);
```

Comment:
Please, remove the console.log

Reply:
Please remove the consoles
![image](https://user-images.githubusercontent.com/11692751/202666178-a8536cda-3ad6-48c3-af4d-2092a251dc8f.png)